x/inflation/keeper/hooks_test.go

```diff
@@ -169,7 +198,7 @@ func (suite *KeeperTestSuite) TestPeriodChangesAfterEpochEnd() {
 			suite.app.InflationKeeper.SetPeriod(suite.ctx, uint64(tc.currentPeriod))
 			currentSkippedEpochs := suite.app.InflationKeeper.GetSkippedEpochs(suite.ctx)
 			currentPeriod := suite.app.InflationKeeper.GetPeriod(suite.ctx)
-			epochIdentifier := suite.app.InflationKeeper.GetEpochIdentifier(suite.ctx)
+			epochIdentifier := tc.epochIdentifier // suite.app.InflationKeeper.GetEpochIdentifier(suite.ctx)
```

Comment:
would remove this line (including comment) and use `tc.epochIdentifier` directly. 

E.g.
```
suite.app.EpochsKeeper.BeforeEpochStart(futureCtx, tc.epochIdentifier, tc.height)
```